ts/src/kraken.ts

```diff
@@ -669,15 +674,20 @@ export default class kraken extends Exchange {
             // see: https://support.kraken.com/hc/en-us/articles/201893608-What-are-the-withdrawal-fees-
             // to add support for multiple withdrawal/deposit methods and
             // differentiated fees for each particular method
-            const code = this.safeCurrencyCode (this.safeString (currency, 'altname'));
+            const altname = this.safeString (currency, 'altname');
+            if (altname !== id) {
+                // only allow for them which has different altname (XBT) and id (XXBT)
+                this.options['sliceAllowedForCurrencyIds'].push (id);
+            }
+            const code = this.safeCurrencyCode (id);
             const precision = this.parseNumber (this.parsePrecision (this.safeString (currency, 'decimals')));
             // assumes all currencies are active except those listed above
-            const active = !this.inArray (code, this.options['inactiveCurrencies']);
+            const active = this.safeString (currency, 'status') === 'enabled';
```

Comment:
they added new field in api response, so we can directly use this instead of hardcoded list